TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/datatypes/UniqueRequestKey.java

```diff
@@ -23,12 +21,12 @@ public int getReplayerRequestIndex() {
 
     @Override
     public String toString() {
-        return trafficStreamKeyAndOffset.getTrafficStreamKey() + "." + getSourceRequestIndex() +
-                (trafficStreamKeyAndOffset.getRequestIndexOffsetAtFirstObservation() == 0 ? "" :
-                        "(offset: "+trafficStreamKeyAndOffset.getRequestIndexOffsetAtFirstObservation()+")");
+        return trafficStreamKey + "." + getSourceRequestIndex() +
+                (sourceRequestIndexOffsetAtFirstObservation == 0 ? "" :
+                        "(offset: "+sourceRequestIndexOffsetAtFirstObservation+")");
```

Comment:
nit: offset is a little confusing to me here I would have expected initial although I may be missing the value here. I originally though this was the number of indices from the `sourceRequestInddexOffsetAtFirstObservation` so I guess I thought this was (getSourceRequestIndex() - sourceRequestInddexOffsetAtFirstObservation)